### ISSUE TITLE
Possibility to Export databases and content in other formats

### ISSUE THREAD

**Issue by catmidia** [Pabloandumundu/Tagstoo#3]
I know you can export your database in JSON format (yes you can put it the name and extension you want but is JSON format (you would be able to explain it better when you export a database )...but what about other formats like CSV or SQL?

**Comment by Pabloandumundu** [comment]
You are right, I can explain it better in the program manual and I will do it in the next version, thanks for the suggestion. Respect to export it in other formats… Take in consideration that the program is home/amaterur oriented and the importance for the program is to be easy to use, in future versions (when I have time and the weather accompany) I will improve interface and usability but the option that you request I found it non-amateur oriented and I must say that there are a lot of utilities that transform json in the format you want with all configurations that you want, as example a search in google has taken me to http://convertjson.com/json-to-sql.htm
